aws/offsite/prune-aurora-backups/README.md

```diff
@@ -27,22 +27,38 @@ sam local invoke PruneAuroraBackupsFunction --event event.json --profile <aws cl
 
 ## Deployment
```

Comment:
could this be a script?

Reply:
Yup. As with the other feature, with this iteration i'm just making the manual instructions more precise. Next time we touch this, we should automate the build/test/deploy.